winlogbeat/make.bat

```diff
@@ -1,19 +1,18 @@
-REM Batch script to build and test on Windows. You can use this in conjunction
-REM with the Vagrant machine.
-REM 
-REM If running inside of Vagrant do this first:
-REM mkdir C:\Gopath\src\github.com\elastic
-REM mklink /d C:\Gopath\src\github.com\elastic\winlogbeat \\vboxsvr\vagrant
-
-REM This is already done inside the Vagrant box.
-REM set PATH=%PATH%;%GOPATH%\bin
-
-go get github.com/tools/godep
-godep go build
-godep go test -race ./...
-
-REM Coverage report:
-REM godep go test -c -cover -coverpkg ./...
-
-godep go test -c -covermode=atomic -coverpkg ./...
-nosetests -w tests\system --process-timeout=30
+REM Batch script to build and test on Windows. You can use this in conjunction
```

Comment:
I guess line terminators changed here. @andrewkroh, can this be an issue or it doesn't matter?